{
	"schema_version": "1.6.0",
	"id": "MAL-2025-990",
	"modified": "2025-02-03T17:07:57Z",
	"published": "2025-02-03T17:07:57Z",
	"summary": "Malicious code in syscontrol (PyPI)",
	"details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
	"affected": [
		{
			"package": {
				"ecosystem": "PyPI",
				"name": "syscontrol",
				"purl": "pkg:pypi/syscontrol"
			},
			"versions": [
				"0.1",
				"0.2"
			],
			"database_specific": {
				"source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/syscontrol/MAL-2025-990.json"
			}
		}
	],
	"credits": [
		{
			"name": "ReversingLabs",
			"contact": [
				"https://www.reversinglabs.com"
			],
			"type": "FINDER"
		}
	],
	"database_specific": {
		"malicious-packages-origins": [
			{
				"id": "RLMA-2025-00531",
				"import_time": "2025-02-03T18:38:09.694269105Z",
				"modified_time": "2025-02-03T17:07:57Z",
				"sha256": "27570a0efd1a7cfc96c9c28cb88f8b2393e85e7eb7601d0e6a13ec3984b78095",
				"source": "reversing-labs",
				"versions": [
					"0.1",
					"0.2"
				]
			}
		]
	}
}
